### ISSUE TITLE
Dynamic Subtitles

### ISSUE THREAD

**Issue by luisbc92** [raycast/extensions#1072]
I would like for extension commands to be capable of displaying a dynamic subtitle, the same way that inline script commands can. This allows an extension to display information to the user at a glance (ex. BitWarden script command can tell the user if it is currently Locked/Unlocked).

An ideal implementation would allow for defining a function such as:
```ts
export function subtitle() {
    return 'Dynamic Subtitle!';
}
```

Which could then be referenced from **package.json** as:
```json
"commands": [
    {
        "name": "index",
        "title": "Test Command",
        "description": "Command with a dynamic subtitle.",
        "subtitle": "${subtitle()}",
        "mode": "view"
    }
]
```

**Describe the feature and the current behavior/state.**
Currently, this feature only exists for script commands, not in the extension API.

**Will this change the current API? How?**
Yes, it will add new functionality.

**Who will benefit from this feature?**
Extension developers who want to provide status information.

**Any Other info.**
No, thanks.

**Comment by luisbc92** [comment]
That's great to know! Thanks for the quick response!

**Comment by mathieudutour** [comment]
it's now available since 1.38.1
